### ISSUE TITLE
TextField text overflow behaves wildly in FireFox

### ISSUE THREAD

**Issue by ujoni** [vaadin/vaadin-text-field#335]
### Description
- Added a TextField using 
  ```java
  TextField field = new TextField("test");
  add(field);
  ```
- used `jetty:run`
- wrote "this is fine" times 3 in the text field

### Expected outcome
Text field with "this is fine this is fine this is fine".

### Actual outcome
![this_is_fine](https://user-images.githubusercontent.com/45562391/53491505-472f3d00-3a9f-11e9-8e58-ec5b2444e4f4.gif)

### Browsers Affected
- [ ] Chrome (72: no)
- [x] Firefox (tested on 65.0.1)
- [ ] Safari
- [ ] Edge (44.17763.1.0/18.17763: no)
- [ ] IE 11
- [ ] iOS Safari
- [ ] Android Chrome

Present in both `V13.0.0.beta1` and `V13.0.0.beta2` - both version use text-field `2.3.2`.

**Comment by edis95** [comment]
I have the same problem and i found how to resolve it with 2 ways:
1. Add placeholder for text field;
1. Add background for "value" with css in theme modules (dom-module) e.g. :
```
[part~="value"]:empty{
background: white !important;
}
```

**Comment by web-padawan** [comment]
Confirmed and still reproducible with Firefox 70 😱

**Comment by web-padawan** [comment]
The problem seems to be a rendering issue and goes away when I comment out `mask-image` usage:

https://github.com/vaadin/vaadin-text-field/blob/6d5f3742b5402887dc908a32a859d1c1d366d42f/theme/lumo/vaadin-text-field-styles.html#L44

https://github.com/vaadin/vaadin-text-field/blob/6d5f3742b5402887dc908a32a859d1c1d366d42f/theme/lumo/vaadin-text-field-styles.html#L63

@jouni do you have an idea how to workaround that?

**Comment by jouni** [comment]
Sigh, I was expecting that this is most probably caused by the mask-image (even before reading Serhii’s comment).

The mask-image seems to be causing more trouble than what it’s worth. I’m considering if we should just drop it.

Though, I thought it should be disabled when the field is focused? It doesn’t look like it is in the screen recording.
https://github.com/vaadin/vaadin-text-field/blob/6d5f3742b5402887dc908a32a859d1c1d366d42f/theme/lumo/vaadin-text-field-styles.html#L47-L52

**Comment by web-padawan** [comment]
Gotcha. Seems like this selector should be split in 2 parts, or else it does not apply 😮

**Comment by steffen-harbich-itc** [comment]
Is this fix already released? I am experiencing the problem with Vaadin Flow 14.0.15 as well as 14.1.1.

**Comment by web-padawan** [comment]
This is released and should be included to 14.1.1 but I will double check.

**Comment by steffen-harbich-itc** [comment]
> 
> 
> This is released and should be included to 14.1.1 but I will double check.

I'm sorry. This is working in 14.1.2 for me.
